eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -276,104 +324,98 @@ transaction_input =
 transaction_output :: Rule
 transaction_output =
   comment
-    [here|
-    Both of the Alonzo and Babbage style TxOut formats are equally valid
-    and can be used interchangeably
-  |]
+    [str|Both of the Alonzo and Babbage style TxOut formats are equally valid
+        |and can be used interchangeably
+        |]
     $ "transaction_output"
-      =:= pre_babbage_transaction_output
-      / post_alonzo_transaction_output
+      =:= shelley_transaction_output
+      / babbage_transaction_output
 
-pre_babbage_transaction_output :: Rule
-pre_babbage_transaction_output =
-  "pre_babbage_transaction_output"
-    =:= arr
-      [ a address
-      , "amount" ==> value
-      , opt ("datum_hash" ==> datum_hash)
-      ]
+shelley_transaction_output :: Rule
+shelley_transaction_output =
+  "shelley_transaction_output"
+    =:= arr [a address, "amount" ==> value, opt ("datum_hash" ==> hash32)]
```

Comment:
It looks like datum_hash definition is not not used if you define this directly `hash32` - i don't know which is better, but if we keep it like this , we could remove [the definition ](https://github.com/IntersectMBO/cardano-ledger/blob/81ef0ce5cb997ee20c5e073bcf9a45cb4f696da2/eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs#L889)